### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by Hartwich75**
I have added navigation and navigation changes between all the current views, so they should work as expected. I have also added the same 3 button structure to the EditExerciseView to match the other views

### CODE REVIEW

File: src/main/java/Group15/View/WorkoutView.java [hunk 1]
```diff
@@ -121,9 +121,8 @@ private static Pane createButtonPane(Workout workout){
             newButton.setPrefSize(200, 50);
             newButton.setOnAction(_ -> {
                 switch (button){
-                    case "Back" -> ViewController.setScene(HomeScreenView.createScene());
-                    case "Edit Workout" -> {
-                    }
+                    case "Back" -> ViewController.setScene(MuscleSelectionView.createMuscleSelectorScene());
```
Comment (WettyWednesday) [inline src/main/java/Group15/View/WorkoutView.java:124]: You are not supposed to go back to the muscleselection from WorkoutView

File: src/main/java/Group15/View/MuscleSelectionView.java [hunk 2]
```diff
@@ -48,7 +48,7 @@ public static Scene createMuscleSelectorScene() {
 
         Button backButton = new Button("Back");
         backButton.setPrefSize(200, 50);
-        backButton.setOnAction(_ -> ViewController.goBack());
+        backButton.setOnAction(_ -> ViewController.setScene(HomeScreenView.createScene()));
```
Comment (WettyWednesday) [inline src/main/java/Group15/View/MuscleSelectionView.java:51]: You dont have to use the setscene here. The only way to get into MuscleSelection is from the homescreen. GoBack method is sufficient

File: src/main/java/Group15/View/EditWorkoutView.java [hunk 3]
```diff
@@ -25,18 +25,19 @@ public class EditWorkoutView
 
     public static Scene createScene(Workout workout)
     {
-        VBox layout = new VBox();
-        layout.setSpacing(20);
-        layout.setAlignment(Pos.CENTER);
+        BorderPane layout = new BorderPane();
```
Comment (WettyWednesday) [inline src/main/java/Group15/View/EditWorkoutView.java:28]: Can you explain your use of BorderPane, instead of a VBox?

**Review by WettyWednesday — CHANGES_REQUESTED** [review]

File: src/main/java/Group15/View/EditWorkoutView.java (hunk 3 above)
Comment (Hartwich75) [reply src/main/java/Group15/View/EditWorkoutView.java:28]: BorderPane is used to divide the layout into different sections and therefore I can set the bottom section as the same for every view

**Review by emilsavino — APPROVED** [review]

**Review by WettyWednesday — APPROVED** [review]

**Review by FredBas — APPROVED** [review]

**Pull request merged**